### DISCUSSION TITLE
Why are OPP-030 through 034 children of LotTender?

### DISCUSSION THREAD

**Discussion by jpmckinney** [OP-TED/eForms-SDK#248] [Q&A] [answered]
LotTender represents a tender (bid). These OPP fields are:

- 030 Contract conditions Code – uses the [contract-detail](https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/contract-detail) authority table
- 031 Contract Conditions Description (other than revenue allocation)
- 032 Revenues Allocation
- 033 Penalties and Rewards Code – uses the [rewards-penalties](https://github.com/OP-TED/eForms-SDK/blob/main/codelists/rewards-penalties.gc#L121) codelist, which seems to be a single-code codelist
- 034 Penalties and Rewards Description

I'm confused as to why these are attached to the bid, and not the lot. Does the bidder decide their own contract conditions, penalties, and rewards? I thought it was the buyer or procuring entity who did so, in which case it seems inappropriate for these fields to hang off individual bids. Can you clarify?

Edit: Same for OPP-080 Kilometers Public Transport. This corresponds to "Kilometres of public passenger transport services" under "V.2.4) Information on value of the contract" on the old [T02 form](https://simap.ted.europa.eu/documents/10184/49059/t02_en.pdf). This field describes the *contract* (or the award), but not the bid.

Edit2: OPP-032 Revenues Allocation seems to correspond to "III.1.3) Allocation of revenues of sales from tickets" on the old T02 form, and OPP-033 and OPP-034 seem to correspond to "III.2.2) Information on rewards and penalties". These appear in section III, and so would be not at all about an individual bid...

**Comment by jpmckinney** [comment]
Pinging @YvesJo perhaps?

**Comment by YvesJo** [comment, marked as answer]
Hi James,
This is a purely business related question that should also be addressed together with business people, therefore I invite you to address it to the Helpdesk instead.
Thanks for your understanding.
